pkg/providers/cloudstack/config/secret.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: eksa-system
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: eksa-license
```

Comment:
I'm missing something here. Why does the eksa license need to be in the cloudstack specific code?